src/spritekit.cs

```diff
@@ -2372,5 +2558,372 @@ interface SKReferenceNode {
 		[Export ("resolveReferenceNode")]
 		void Resolve ();
 	}
+
+	[iOS (9,0)][Mac (10,11)]
+	[BaseType (typeof(NSObject))]
+	interface SKAttribute : NSCoding
+	{
+		[Static]
+		[Export ("attributeWithName:type:")]
+		SKAttribute Create (string name, SKAttributeType type);
+
+		[Export ("initWithName:type:")]
+		[DesignatedInitializer]
```

Comment:
Add `[DisableDefaultCtor]` when a `[DesignatedInitializer]` is present. It's likely unusable and, if we're wrong, it's easier to add later than remove it.